app/services/supplementary_data.py

```diff
@@ -121,14 +126,19 @@ def decrypt_supplementary_data(
 
 
 def validate_supplementary_data(
-    supplementary_data: Mapping, dataset_id: str, identifier: str, survey_id: str
+    supplementary_data: Mapping,
+    dataset_id: str,
+    identifier: str,
+    survey_id: str,
+    sds_schema_version: str | None = None,
 ) -> dict:
```

Comment:
Little detail here but for the return value type hint we recommend in our guide to at least give types of the dict keys and values.

Reply:
Updated these 👍